src/components/shared/ButtonWithPanel/ConfirmDialog.css

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is mostly resurrected from what was removed in ArrowPanel in previous commits.